app/processing/requires-manual-ledger-check.js

```diff
@@ -15,7 +16,8 @@ const requiresManualLedgerCheck = async (paymentRequest) => {
   }
 
   const previousPaymentRequests = await getCompletedPaymentRequests(paymentRequest)
-  return previousPaymentRequests.length && previousPaymentRequests.some(x => x.value < 0)
+  const paymentRequestsToCheck = ignoreZeroValueSplits(previousPaymentRequests)
+  return paymentRequestsToCheck.length > 0 && paymentRequestsToCheck.some(x => x.value < 0)

```

Comment:
Don't think you need `> 0` here?

Reply:
Don't _reaaally_ need it, but a curious point is if the length is 0, the statement actually returns `0` instead of `false`.

Programmatically doesn't make a huge difference, but was throwing one of the tests I introduced.